contentcuration/contentcuration/models.py

```diff
@@ -199,6 +199,7 @@ class Channel(models.Model):
     description = models.CharField(max_length=400, blank=True)
     version = models.IntegerField(default=0)
     thumbnail = models.TextField(blank=True, null=True)
+    thumbnail_encoding = models.TextField(blank=True, null=True)
```

Comment:
thumbnail refers to the actual file, thumbnail_encoding refers to the formatted base64 encoding that allows users to recenter/crop thumbnails